elasticdl/python/tests/worker_mnist_test.py

```diff
@@ -226,6 +227,45 @@ def test_compare_mnist_train(self):
         for w, l in zip(worker_results, local_results):
             self.assertTupleEqual(w, l)
 
+    def test_worker_pull_embedding(self):
```

Comment:
Maybe we move this test to worker_test.py? worker_mnist_test is for testing worker training mnist dataset.

Reply:
I forgot to rename this file. I put this test here because I want to rename this file to test interaction between workers and PS instances. Otherwise, there will be another test file worker_frappe_test.py (frappe is a recommending dataset), and these two files will have many repeated code.

Reply:
Rename it to worker_and_PS_interaction_test.py.